tasks/emoji-unicode-version-map/index.js

```diff
@@ -37,83 +10,33 @@ const aliasMap = {
 
 
 function generateEmojiUnicodeVersionMap() {
-	const addingEmojipediaUnicodeVersionPromises = Object.keys(emojipediaUnicodeVersionUrlMap).map((unicodeVersionKey) => {
-		console.log(`Working on ${unicodeVersionKey}...`);
-		const emojipediaUrl = emojipediaUnicodeVersionUrlMap[unicodeVersionKey];
-		const cacheFileName = sanitizeForFileName(emojipediaUrl);
-		const cacheFilePath = `./cached-pages/${cacheFileName}.html`;
-		// Try to read from the cache
-		return readFile(cacheFilePath, 'utf8')
-			// Fallback to fetching the page
-			.catch(() => {
-				return fetch(emojipediaUrl)
-					.then((res) => res.text())
-					.then((html) => {
-						// Save the file as a cache
-						outputFile(cacheFilePath, html);
-						return html;
-					});
-			})
-			.then((html) => {
-				scrapeEmojipediaHtml(emojiMap, html, unicodeVersionKey);
-			});
-	});
+	const emojiToUnicodeVersionMap = {};
 
-	// Save the transformed emoji map
-	const fillingZwjSequencesPromise = Promise.all(addingEmojipediaUnicodeVersionPromises)
-		.then(() => {
-			console.log('Filling out ZWJ sequences...');
-			fillZwjSequenceUnicodeVersions(emojiMap);
-		});
+	Object.keys(emojioneMap).forEach((emojioneKey) => {
+		const emoji = emojioneMap[emojioneKey];
 
-	// Setup the legacy aliases
-	const addingAliasesPromise = fillingZwjSequencesPromise
-		.then(() => {
-			Object.keys(aliasMap).forEach((aliasName) => {
-				const aliasReferenceName = aliasMap[aliasName];
-				emojiMap[aliasName] = Object.assign({}, emojiMap[aliasReferenceName]);
-			});
-		});
+		const emojiShortName = emoji.shortname;
+		const emojiNameKey = emojiShortName.slice(1, emojiShortName.length - 1);
```

Comment:
We can just use the `emoji.name` AFAICT

Reply:
not if you want it to be compatible with emojione short names (`name`  is the Unicode name AFAICT)

Reply:
confirmed. For instance, one of them has the name `palms up together: dark skin tone` while its emojione short name is `:palms_up_together_tone5:`

Reply:
Thanks for looking into it!